.github/workflows/build.yml

```diff
@@ -16,6 +16,9 @@ jobs:
             - name: Install Dependencies
               run: yarn install --frozen-lockfile
 
+            - name: Install chakra-ui types
```

Comment:
This script generates the custom chakra-ui variants. This is now a prerequisite for a successful build as otherwise we'll get ts errors for invalid variants.